server/src/main/scala/services/exports/Exports.scala

```diff
@@ -29,6 +29,13 @@ import scala.concurrent.{ExecutionContext, Future}
 object Exports {
   val log: Logger = LoggerFactory.getLogger(getClass)
 
+  val acl_arrival_status = ArrivalStatus("ACL Forecast")
+
+  val passengerFlights: Arrival => Boolean = apiFlight =>
```

Comment:
this should be called `passengerFlightsFilter`

Reply:
we can stick some unit tests around it as well maybe.

Reply:
I have added e2e test but I can also some unit here